### ISSUE TITLE
[Suggestion] Add support for Just Giving donations

### ISSUE THREAD

**Issue by rshackleton** [iGoodie/TwitchSpawn#28]
Streamlabs supports a UK charity platform called Just Giving, it would be great to have notifications be triggered from donations.

The event data from the log file is:

```
[16:42:59] [EventThread/INFO]: Received Streamlabs packet (justgivingdonation, justgiving) -> {"amount":99,"isTest":true,"name":"Shackleberry112","currency":"GBP","from":"John","to":{"name":"Shackleberry112"},"_id":"f3d16e85636b8e2fd698b1830d97a427","message":"This is a test donation for £99.00.","priority":10,"formatted_amount":"£99.00"}
```

```json
{
  "amount": 99,
  "isTest": true,
  "name": "Shackleberry112",
  "currency": "GBP",
  "from": "John",
  "to": { "name": "Shackleberry112" },
  "_id": "f3d16e85636b8e2fd698b1830d97a427",
  "message": "This is a test donation for £99.00.",
  "priority": 10,
  "formatted_amount": "£99.00"
}

```

**Comment by iGoodie** [comment]
Thanks once again for sharing the event data!
I'll be adding `JustGiving Donation` in the next update! 🎉

**Comment by iGoodie** [comment]
It shall now be implemented with 5fce47d ✌️ 😊
